src/foam/nanos/crunch/Capability.js

```diff
@@ -313,6 +313,13 @@ foam.CLASS({
       view: {
         class: 'foam.u2.view.ReferenceArrayView'
       }
+    },
+    {
+      class: 'Boolean',
+      name: 'disableAutoSave',
```

Comment:
Since I expect to see autoSave , how about renaming this to autoSave and defaulting the value to 'true'.